test/audits/conversion.test.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+import chai from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import nock from 'nock';
+import { conversionAuditRunner } from '../../src/conversion/handler.js';
+import { rumDashboardData, rumSourcesData, expectedConversionData } from '../fixtures/conversion-data.js';
+
+chai.use(sinonChai);
+const { expect } = chai;
+
+const DOMAIN_REQUEST_DEFAULT_PARAMS = {
+  interval: 7,
+  offset: 0,
+  limit: 101,
+};
+describe('Conversion Audit', () => {
+  let context;
+  let mockLog;
+
+  beforeEach('setup', () => {
+    mockLog = {
+      info: sinon.spy(),
+      warn: sinon.spy(),
+      error: sinon.spy(),
+    };
+    context = {
+      log: mockLog,
+      env: {
+        RUM_DOMAIN_KEY: 'domainkey',
+      },
+    };
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+    sinon.restore();
+  });
+
+  it('fetch conversion data for base url > process > send results', async () => {
+    nock('https://www.spacecat.com')
+      .get('/')
+      .reply(200);
+    nock('https://helix-pages.anywhere.run')
```

Comment:
Here I would mock the RumApiClient, and check the parameters is calledWith, instead of using nock

Reply:
Done.

Reply:
Alina, I have done the changes as per our discussion on call.